djmoney/contrib/exchange/backends/base.py

```diff
@@ -45,9 +45,12 @@ def get_params(self):
         return {}
 
     def get_response(self, **params):
-        url = self.get_url(**params)
+        headers = {
+            "User-Agent": "Mozilla/5.0",
```

Comment:
You could get into trouble for faking the User-Agent, not sure. But I'll leave that up to others to scrutinize :)